Mage/src/main/java/mage/game/permanent/PermanentToken.java

```diff
@@ -25,8 +26,9 @@ public PermanentToken(Token token, UUID controllerId, String expansionSetCode, G
         this.token = token.copy();
         this.token.getAbilities().newOriginalId(); // neccessary if token has ability like DevourAbility()
         this.token.getAbilities().setSourceId(objectId);
-        this.power.modifyBaseValue(token.getPower().getBaseValueModified());
-        this.toughness.modifyBaseValue(token.getToughness().getBaseValueModified());
+        // TODO: Should this be using base or modified toughness?
```

Comment:
?

Reply:
@awjackson thoughts?

Reply:
If I'm not mistaken, the 'token' argument to this constructor is always a template and not an actual permanent, so again it shouldn't matter.